packaging/pep517_backend/_backend.py

```diff
@@ -379,7 +379,7 @@ def get_requires_for_build_wheel(
     elif sysconfig.get_config_var('Py_GIL_DISABLED'):
         c_ext_build_deps = ['Cython ~= 3.1.0a1']
```

Comment:
e.g. This can be removed:
```suggestion
```

If you could also check through https://github.com/aio-libs/propcache/pull/84/files and remove anything else (at a glance, looks like we need to remove a block in pyproject.toml and delete a requirements file atleast).